index.css

```diff
@@ -1,8 +1,59 @@
+:root {
+  --accent-color: rgb(67, 155, 245);
+  --current-color: #adbac2;
+
+  --body-bg-color: rgb(34, 39, 46);
+
+  --header-bg-color: rgba(45, 51, 59, 0.3);
+
+  --hero-bg-img-color: linear-gradient(rgba(47, 48, 58, 0.4), rgba(47, 48, 58, 0.4));
```

Comment:
this is a gradient, not color. separate colors or rename variable

Reply:
change in mobile-menu branch